### ISSUE TITLE
SQLPro for SQLite pboxd crashes

### ISSUE THREAD

**Issue by jmadren** [hankinsoft/SQLPro#782]
I get a crash message/report occasionlly saying pboxd has crashed, supposedly because of something SQLPro did. This happens almost every day that I use SQLPro for SQLite. I just Ok/dismiss the message and everything appears to continue working fine. But it happens often enough that I figured I should report it.

A sample crash log is attached
[pboxd crash log.zip](https://github.com/hankinsoft/SQLPro/files/8672352/pboxd.crash.log.zip)
.

**Comment by hankinsoft** [comment]
Are you using SQLPro to modify some system databases? Thats the only reason I can think that something other than SQLPro could crash and report SQLPro as the reasoning.

**Comment by jmadren** [comment]
I suppose you mean system tables? No, not modifying system tables. I used to modify sequence tables, which I guess is considered a system table, but I haven't done that in quite a while, mainly because I removed all AUTOINCREMENT options on all tables.

Of course, whenever I modify the structure of a table, that results in modification of sqlite_master, but sqlite is doing that itself. But I have been adding/modifying table structures quite a bit (still in early development stage of a project). Maybe that has something to do with it?
